yt_dlp/utils.py

```diff
@@ -5876,6 +5876,153 @@ def orderedSet_from_options(options, alias_dict, *, use_regex=False, start=None)
     return orderedSet(requested)
 
 
+class InitializationFragmentError(YoutubeDLError):
+    def __init__(self):
+        super().__init__(
+            'Initialization fragment found after media fragments, unable to download')
```

Comment:
Maybe its better this way because I use it in the try, except of the downloader

```python
        try:
            fragments = media_manifest.get_fragments(info_dict.get('format_index'), ctx['fragment_index'], extra_query)
        except InitializationFragmentError as e:
            self.report_error(e.msg)
            return False
```